src/Card.css

```diff
@@ -20,7 +20,7 @@
     color: #FFFDF8;
     padding: 10px;
     width: 230px;
-    margin: none;;
+    margin: none;
```

Comment:
Let's look at editing the styles so the long titles don't cut off the rest of the details pop up